source/config.js

```diff
@@ -104,6 +106,10 @@ const defaultConfig = {
   // "pluginConfigs": { "gerrit": { "disabled": true } }
   pluginConfigs: {},
 
+  // full path to git executable. If not set, system default will be used
+  // (in this case, git should be available on PATH)
+  gitCommand: null,
```

Comment:
It might be also useful to add it as an option - for example, in [Brackets](http://brackets.io) editor one could use setting from `brackets-git` extension config and pass it over to ungit, instead of setting it via `.ungitrc`. Should we have that added?